tjreports/site/views/reports/view.html.php

```diff
@@ -98,8 +98,15 @@ public function display($tpl = null)
 		// Get all vendars from backend
 		if (empty($client))
 		{
-			$params = JComponentHelper::getParams('com_tjreports');
-			$client = $params->get('vendars');
+			$params = $app->getParams();
+			$client = $params->get('clients');
+
+			// Check for multiple clients
+			if (is_array($client))
+			{
+				$client = implode(",", $client);
+			}
+
 			$input->set('client', $client);
```

Comment:
@hemantpurohit1  why the client is pushed in the input? Also, the following code, can this be written in model's populatestate function?

$params = $app->getParams();
$client = $params->get('clients');